dashboard-frontend/src/helpers/formBuilder.ts

```diff
@@ -0,0 +1,57 @@
+import Text from '../pages/NewStudyPage/components/form/inputs/Text';
+import Select from '../pages/NewStudyPage/components/form/inputs/Select';
+import { ConfBase, FieldBase } from '../types/form';
+import { createNameFor } from './strings';
+
+const str: keyof FieldBase = 'type';
+
+export const formBuilder = (field: FieldBase, localFormData?: any) => {
```

Comment:
I don't love the thing called `formBuilder` builds a field, not a form... Another naming issue ;)

Reply:
Renamed to `translateField`, I think that makes more sense